dev-libs/nmeap/nmeap-0.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Extensible NMEA-0183 (GPS) data parser in standard C"
+HOMEPAGE="http://nmeap.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test-programs"
+
+DEPEND="doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	default
+
+	# Repsect users CFLAGS and LDFLAGS for the static lib archive
+	sed -i -e 's/CFLAGS =/CFLAGS +=/' -e 's/-g -O0 -Werror -Wall/$(LDFLAGS)/' src/Makefile || die
+
+	# Respect users CFLAGS and LDFLAGS for the test programs
+	sed -i -e 's/-g -O0//' -e 's/-Wall -Werror/$(CFLAGS) $(LDFLAGS)/' tst/Makefile || die
+
+	# Only build those test programs, if a user wants it
+	if ! use test-programs; then
+		sed -i -e '/TST/d' Makefile || die
+	fi
+
+	# Silent output of Doxygen and update it, since it is quite old
+	if use doc; then
+		sed -i -e 's/QUIET.*/QUIET = YES/' Doxyfile || die
+		doxygen -u Doxyfile 2>/dev/null || die
+	fi
+}
+
+src_install() {
+	dolib.a lib/libnmeap.a
+
+	doheader inc/nmeap.h inc/nmeap_def.h
+
+	if use doc; then
+		local HTML_DOCS=( "doc/tutorial.html" "doc/html" )
+		doxygen Doxyfile || die
+	fi
+
+	if use test-programs; then
```

Comment:
Do you install any documentation that explains the purpose of those programs? If not, then installing programs called `nmeap-testN` looks like a useless idea.

Reply:
They generate different nmea output, if someone needs fake gps. But for my understanding, this is only for testing purposes. What do you think, remove it? For LCD4Linux, for this dependency it is, it's not needed.

Reply:
I would not install something that's not documented and has no clear use for the user.

Reply:
Ok, fine for me. I dropped it now.